packages/target-electron/build/afterPackHook.cjs

```diff
@@ -102,8 +102,10 @@ module.exports = async context => {
 
   // copy map xdc
   // ---------------------------------------------------------------------------------
-  if (!env['NO_ASAR']) {
-    await copyMapXdcToUnpackedASAR(resources_dir, source_dir)
+  if (env['NO_ASAR']) {
+    await copyMapXdc(resources_dir, source_dir, false)
+  } else {
+    await copyMapXdc(resources_dir, source_dir, true)
   }

```

Comment:
```suggestion
  const asar = env['NO_ASAR'] ? false : true
  await copyMapXdc(resources_dir, source_dir, asar)
```

Reply:
Mind adding a comment here describing why it looks like it does? What does `NO_ASAR` means? When should this be copied and when shouldn't it be copied?

Reply:
asar is some electron file archive format, no asar means not using it.